TestModels/LanguageSpecificLogic/README.md

```diff
@@ -0,0 +1,59 @@
+# LanguageSpecificLogic
+
+This project demonstrates generating target-language-specific code from Dafny.
+This allows developers to write Dafny code that is only generated and run for a particular target language.
+This allows developers to increase the amount of language-specific code that is verified by Dafny.
```

Comment:
Nit: re-phrase to avoid repetition 
```suggestion
This allows developers to write and **verify** Dafny code 
that is only generated and run 
for a particular target language.
```